### PR TITLE
Removal of premium references

### PR DESCRIPTION
**Pull request by abravosuse**
Not to be published until version 2.4 is released.

Trento 2.4 will remove the premium flag and, therefore, all references to premium will have to be removed from the official documentation.

### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -106,12 +106,11 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
-    <title>What is &trentopremium;?</title>
+    <title>What is &trento;?</title>
     <para>
-      &trentopremium; is the extended version of the Trento community project containing
-      additional checks. Apart from their naming, both are comprehensive monitoring solutions
-      consisting of two main components, the &t.server; and the &t.agent;.
-      Both Trento variants provide the following features:
+      &trento; is the supported version of the open source Trento project. It consists
+      of two main components, the &t.server; and the &t.agent;, and it 
+      provides the following features:
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:113]:
```suggestion
      &trento; is the supported version of the open source Trento project. It consists
      of two main components, the &t.server; and the &t.agent;. &trento; 
      offers the following:
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -219,7 +218,7 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
     <!--
       Most ist taken from https://confluence.suse.com/x/uoOyN
     -->
-    <para>Trento Premium is available at no extra cost for SLES for SAP subscribers. Particularly, Trento's two main components have the following product lifecycles:</para>
+    <para>Trento is available at no extra cost for SLES for SAP subscribers. Particularly, Trento's two main components have the following product lifecycles:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:221]:
```suggestion
    <para>Trento is available at no extra cost for SLES for SAP subscribers.  Trento's two main components have the following product lifecycles:</para>
```

**Review by abravosuse — COMMENTED** [review]
Thanks for the suggestions @dmpop ! All accepted AS-IS.

**Comment by tomschr** [comment]
This is integrated in main in commit 492ee7df628 now.

**Pull request closed without merging**